app/models/student_loans.rb

```diff
@@ -35,4 +37,29 @@ def feedback_url
   def short_name
     I18n.t("student_loans.policy_short_name")
   end
+
+  # Returns the AcademicYear during or after which teachers must have completed
+  # their Initial Teacher Training and been awarded QTS to be eligible to make
+  # a claim. Anyone qualifying before this academic year should not be able to
+  # make a claim.
+  #
+  # Teachers that qualified after 2013 are eligible to claim back student loans
+  # repayments for 10 years. Their first claim will be made in the subsequent
+  # year because they are retrospectively claiming *back* repayments made during
+  # the *financial year*. So for example if you qualify in 2021/2022, you are
+  # eligible to claim back student loan repayments you make in the 2021/2022
+  # "financial year", which ends April 2022, and the claim for that period can
+  # be made from the start of the 2022/2023 "academic year".
+  #
+  # So to give concrete examples, teachers qualifying in 2013/2014 can make
+  # claims up to 2024/2025, and a teacher qualifying in 2014/2015 can make
+  # claims up to 2025/2026 and so on.
+  def first_eligible_qts_award_year
+    eleven_years_prior = AcademicYear.new(configuration.current_academic_year) - 11
```

Comment:
At first I was thinking "why is it 11 here and not 10?" but the comment and examples explained it nicely 👍

Reply:
In the follow-up work in the other branch it gets a name too: https://github.com/DFE-Digital/dfe-teachers-payment-service/compare/1223-vary-qts-eligibility-based-on-academic-year?expand=1#diff-3f76becab371c5075ab8d7f820fa49faR62